pkg/scheduler/algorithm/priorities/resource_limits.go

```diff
@@ -42,7 +42,14 @@ func ResourceLimitsPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *schedule
 	allocatableResources := nodeInfo.AllocatableResource()
 
 	// compute pod limits
-	podLimits := getResourceLimits(pod)
+	podLimits := &schedulernodeinfo.Resource{}
```

Comment:
```suggestion
	var podLimits *schedulernodeinfo.Resource
```

can save the cost of an op :)

Reply:
done. PTAL